Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -378,14 +380,16 @@ def url_command(client: Client, url: str) -> Tuple[str, Dict, Union[Dict, list]]
                 'Reliability': client.reliability
             })
     if not raws:
+        if no_matches_url:
+            return f"No matches for URL\S {no_matches_url}", {}, {}
         return f'{INTEGRATION_NAME} - Could not find any results for given query', {}, {}
     context_entry: dict = {
         outputPaths.get("url"): url_ec,
         'AlienVaultOTX.URL(val.Url && val.Url === obj.Url)': alienvault_ec,
         outputPaths.get("dbotscore"): dbotscore_ec
     }
-    human_readable = tableToMarkdown(t=context_entry.get('AlienVaultOTX.URL(val.Url && val.Url === obj.Url)'),
-                                     name=title)
+    table = tableToMarkdown(t=context_entry.get('AlienVaultOTX.URL(val.Url && val.Url === obj.Url)'), name=title)
+    human_readable = f"{table} No matches for URL\S {no_matches_url}" if no_matches_url else table
```

Comment:
Is this still valid even with one URL is given with no results? Please provide screenshots of the following cases : 
1. One URL is given without results
2. Two URLs are given - First one with no results, second with results
3. Two URLs are given - First one with results, second without results

Reply:
[Uploading without results.zip…]()
[Uploading the second without results.zip…]()

Reply:
<img width="1582" alt="the first without resutls" src="https://user-images.githubusercontent.com/78307768/116046428-bc94f900-a67b-11eb-9fa6-0643403a323f.png">

Reply:
<img width="1581" alt="without results" src="https://user-images.githubusercontent.com/78307768/116046497-cfa7c900-a67b-11eb-868f-b5d65ba31787.png">
<img width="1588" alt="the second without results" src="https://user-images.githubusercontent.com/78307768/116046509-d3d3e680-a67b-11eb-9f43-f14c05cc94e0.png">